packages/core/src/routes/logto-config.ts

```diff
@@ -287,4 +290,55 @@ export default function logtoConfigRoutes<T extends AuthedRouter>(
       return next();
     }
   );
+
+  if (!EnvSet.values.isCloud) {
+    return;
+  }
+
+  router.post(
+    '/configs/jwt-customizer/test',
+    koaGuard({
+      body: z.discriminatedUnion('tokenType', [
+        z.object({
+          tokenType: z.literal(LogtoJwtTokenKey.AccessToken),
+          payload: accessTokenJwtCustomizerGuard,
+        }),
+        z.object({
+          tokenType: z.literal(LogtoJwtTokenKey.ClientCredentials),
+          payload: clientCredentialsJwtCustomizerGuard,
+        }),
+      ]),
+      response: jsonObjectGuard,
+      /**
+       * Code 400 indicates Zod errors in cloud service (data type does not match expectation, can be either request body or response body).
+       * Code 422 indicates syntax errors in cloud service.
+       * Code 500 indicates internal server errors in cloud service.
```

Comment:
actually should we put this in the swagger file?

Reply:
Put this into swagger file results to error since it can not properly filter out cloud only routes for OSS version and throws error. Do not add this API path to swagger file for now.